src/server/server.ts

```diff
@@ -141,6 +141,25 @@ app.post('/api/make-move', (req, res) => {
     })
 })
 
+app.post('/api/rematch', (req, res) => {
```

Comment:
Better to be explicit here, define two endpoints `/api/requestRematch` and `/api/acceptRematch`

Reply:
Fixed.